neo/Consensus/ConsensusService.cs

```diff
@@ -116,6 +116,11 @@ private void CheckExpectedView(byte viewNumber)
                 if (message is null || message.NewViewNumber < viewNumber)
                     localNode.Tell(new LocalNode.SendDirectly { Inventory = context.MakeChangeView(viewNumber) });
                 InitializeConsensus(viewNumber);
+                // Save View Change without Transactions for restoring after a restart.
+                var savedTransactions = context.Transactions;
+                context.Transactions = null;
```

Comment:
`context.Transactions = new Dictionary<UInt256, Transaction>();`
maybe we can prevent the null value (could be no thread safe if is not checked this null value)

Reply:
The code happens in the same thread. There is no thread safety issue.

Reply:
Also, it handles null in the serialization.

Reply:
In serialization yes, but in other parts of the code don't.